apps/meteor/app/lib/server/functions/saveCustomFieldsWithoutValidation.ts

```diff
@@ -37,7 +39,7 @@ export const saveCustomFieldsWithoutValidation = async function (userId, formDat
 				update.$set[modifyRecordField.field] = customFields[fieldName];
 			}
 
-			await Users.updateOne(userId, update);
+			await Users.updateOne({ _id: userId }, update);
```

Comment:
I guess this was not working? 🤔  it this function being used anywhere? should we add some tests?

Reply:
Yep, was not working 😬 


Let's see if we have tests for it

Reply:
oh, we handle that 💪 

https://github.com/RocketChat/Rocket.Chat/blob/bc22cfa4c08fbdca4963040177b18fb190775704/apps/meteor/server/models/raw/BaseRaw.ts#L160

Reply:
oops, that's on `findOne`, not `updateOne` 🤦

Reply:
how is TS not complaining about it? can `Filter` be a string? 🤔

Reply:
Good question. Maybe one type on mongo side is too broad or smth, cause this is something that happened multiple times as of now (string where Filter is expected) 😬